src/BlockID.h

```diff
@@ -499,6 +537,12 @@ enum
 	E_BLOCK_BED_OCCUPIED = 4,
 	E_BLOCK_BED_BED_HEAD = 8,
 
+	// E_BLOCK_BEETROOTS metas:
+	E_BLOCK_BEEROOTS_AGE_ZERO = 0,
+	E_BLOCK_BEEROOTS_AGE_ONE = 1,
+	E_BLOCK_BEEROOTS_AGE_TWO = 2,
+	E_BLOCK_BEEROOTS_AGE_THREE = 3,
```

Comment:
These are named wrong, we use `E_META_XYZ` for meta values (forgive the errors just above your changes, haven't fixed those in time)

Also, I'm not sure if we need to have an age of `ONE`, `TWO`, `THREE` etc. corresponding to the same number. I believe using the number instead is self-explanatory. So just remove all these.